### ISSUE TITLE
How to retrieve data without knowing it?

### ISSUE THREAD

**Issue by alper** [matthewpalmer/Locksmith#122]
The documentation isn't exactly clear about this. From the example:

```
let account = TwitterAccount(username: "_matthewpalmer", password: "my_password")
let result = account.readFromSecureStore()
```

But say I'm interested in at some point retrieving that password which I stored in the keychain. How would I do that? I can't create a TwitterAccount without it can I?

**Comment by tkeithblack** [comment]
I have the same question. I've been able to successfully use:

try Locksmith.saveData(["some key": "some value"], forUserAccount: "myUserAccount")
let dictionary = Locksmith.loadDataForUserAccount("myUserAccount")

However, I need more control and wanted to use the approach illustrated in the TwitterAccount example. I guess I'm missing something because after writing the object to the keychain and coming back later I was expecting to create a new object 'account = TwitterAccount' an use 'readFromSecureStore()' to grab the username and password that I stored earlier.

Thanks,
KeithB
